### DISCUSSION TITLE
Onnx input size

### DISCUSSION THREAD

**Discussion by AD-HO** [microsoft/onnxruntime#4930] [General]
Hi Everyone,
I would like to know how to unfreeze the input size of an exported Onnx model?
Example: https://pytorch.org/tutorials/advanced/super_resolution_with_onnxruntime.html

Super-Resolution is an image-to-image translation, i can’t see how i would consume this model without resizing the input to 224 x 224 , and this mean the input quality will decrease
thank you.

**Comment by ytaous** [comment]
https://github.com/microsoft/onnxruntime/issues/4929#issuecomment-681198278

**Reply by natke** [reply]
Hi @AD-HO, does this answer your question? If so, can you please mark this as "answer". Thank you!

**Reply by AD-HO** [reply]
not really, i have to try this https://github.com/onnx/onnx/issues/2977#issuecomment-683644884
